app/assets/javascripts/templates/modal.jsx

```diff
@@ -0,0 +1,17 @@
+var Modal = ReactBootstrap.Modal;
+
```

Comment:
List props here too if applicable.

Reply:
No props for `DefaultModal`, but I added a description for clarification